### ISSUE TITLE
hello,i found my loss is always nan.

### ISSUE THREAD

**Issue by TitaniumTTL** [jiwidi/DeepSpeech-pytorch#6]
hello, i run your code to train the model.
i found my loss including test-loss is always Nan. The output of the model is 0 (dim = 2).
"output = F.log_softmax(y_hat+1e-10)" (in line 154 in deepspeech_main.py). It doesn't work that y_hat plus with [eps.[]
![1](https://user-images.githubusercontent.com/43907135/104300782-df066a80-5501-11eb-8946-f9f969a2c17d.png)
